### PR TITLE
fixed <b> tags on scenario reference cards and on traits

### PR DESCRIPTION
**Pull request by robert-huang**
also updated some formatting to be consistent (Deck size -> Deck Size)

### CODE REVIEW

File: pack/dwl/tece_encounter.json [hunk 1]
```diff
@@ -247,7 +247,7 @@
         "position": 172,
         "quantity": 1,
         "shroud": 4,
-        "text": "Sleeping Car is connected to the locations to the left and right of it.\n[action]:<b> </b>Gain 3 resources. Remember that you have \"stolen a passenger's luggage.\" (Group limit once per game).\n",
+        "text": "Sleeping Car is connected to the locations to the left and right of it.\n[action]: Gain 3 resources. Remember that you have \"stolen a passenger's luggage.\" (Group limit once per game).\n",
```
Comment (drawntotheflame) [inline pack/dwl/tece_encounter.json:250]: Please can you remove the rogue '\n' at the end of this line.

File: pack/dwl/tmm.json [hunk 2]
```diff
@@ -138,7 +138,7 @@
         "quantity": 2,
         "skill_willpower": 1,
         "slot": "Arcane",
-        "text": "Uses (5 charges).\n[action]<b> </b>Spend 1 charge: <b>Fight. </b>This attack uses [willpower] instead of [combat]. You get +1 [willpower] for this attack. If a [skull] symbol is revealed during this attack, this attack deals +2 damage.",
+        "text": "Uses (5 charges).\n[action] Spend 1 charge: <b>Fight. </b>This attack uses [willpower] instead of [combat]. You get +1 [willpower] for this attack. If a [skull] symbol is revealed during this attack, this attack deals +2 damage.",
```
Comment (drawntotheflame) [inline pack/dwl/tmm.json:141]: Since we're doing tidying - please can you move the closing bold tag up to right after 'Fight.' with a space afterwards?

**Comment by drawntotheflame** [comment]
It might just be my view settings on github, but it doesn't look like you've removed the bold tags for some of the scenario cards, such as the ones in the core set. Am I missing something?

**Comment by robert-huang** [comment]
hmm, that was like the _first_ thing i did lol, i must have accidentally reverted some stuff when i was trying to fix line endings by messing around with version control

**Comment by robert-huang** [comment]
> Yeah, I'm not sure what that line ending stuff is - it looks like rogue spaces to me?

just checked, there's no spaces at the end of those lines, some of those are just unix-style line endings and other are windows-styles ones - i think there's just a mix of them peppered throughout the repo from various contributors adding stuff from different systems, so when my editor processes them they get detected as changes

**Comment by robert-huang** [comment]
@drawntotheflame can you check this so it can go up?

**Comment by drawntotheflame** [comment]
@Kamalisk I think these can all go up.

Great job @robert-huang

**Comment by robert-huang** [comment]
Thanks :)

**Comment by robert-huang** [comment]
@Kamalisk could you deploy this? Thanks :)

**Pull request merged**